src/components/MediaSettingsModal/MediaSettingsModal.jsx

```diff
@@ -86,7 +86,7 @@ export const MediaSettingsModal = ({
     return onProceed({
       data: {
         ...rest,
-        name: `${name}.${fileExt}`,
+        name: `${name}${fileExt ? `.${fileExt}` : ""}`,
```

Comment:
nit: Can we add a brief comment to explain why this operation is needed for a better readability of the code? 
After reading this line `const fileExt = getFileExt(mediaData?.name || "")`, I feel `fileExt` would be `""` if the `name` of `mediaData` is missing. What edge case does this line address by manually checking `fileExt` and assign `""`?

Reply:
Oh this actually adds a period when fileExt exists - i've added comments in https://github.com/isomerpages/isomercms-frontend/pull/1184/commits/b8b38e5e54164d5519c0cc22bd552f5ed3d23138 to clarify

We'll might want to split up the create and update modals at some point in future, since this behaviour is a little unintuitive